docs/apps/starccm+.md

```diff
@@ -8,7 +8,7 @@ CSC - IT Center for Science has no own Star-CCM+ licenses available for customer
 
 ## Available
 
-Several versions of Star-CCM+ are available on the [Puhti](../computing/systems-puhti.md) server for analysis runs. Use command
+Several versions of Star-CCM+ are available on [Puhti and Mahti](../computing/available-systems.md) servers. Use command
 
     ls /appl/soft/eng/star/ | grep '^[0-9]'
```

Comment:
Command is not valid on Mahti. Perhaps `ls -l /appl/soft/eng/starccm++/ | grep '^[0-9]' ` ?

Reply:
On both platforms the directory name should be starccm+.  I have now changed those names, ie, on both Puhti and Mahti the name was incorrect.